src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/MuTect2.java

```diff
@@ -612,6 +583,19 @@ public ActivityProfileState isActive(final RefMetaDataTracker tracker, final Ref
         return annotateVCs(calledHaplotypes, metaDataTracker);
     }
 
+    @Override
```

Comment:
Maybe add a TODO here to mark that the method is duplicated from HC and that we should do something about it?